### ISSUE TITLE
collected_at ValueError: Value can't be None

### ISSUE THREAD

**Issue by rasmuslp** [Taxel/PlexTraktSync#949]
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists

### The problem

When I scan my Plex server, the process exits with an unhandled exception.

I have successfully authenticated with Plex and Trakt, and if I exclude my `Series` library it runs the scan to completion. 

### Error trace / logs

```text
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/app/plextraktsync/__main__.py", line 18, in <module>
    cli()
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1635, in invoke
    rv = super().invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/app/plextraktsync/cli.py", line 46, in cli
    sync()
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/app/plextraktsync/cli.py", line 28, in wrap
    cmd(*args, **kwargs)
  File "/app/plextraktsync/commands/sync.py", line 60, in sync
    runner.sync(walker=w, dry_run=config.dry_run)
  File "/app/plextraktsync/sync.py", line 75, in sync
    self.sync_collection(episode, dry_run=dry_run)
  File "/app/plextraktsync/sync.py", line 103, in sync_collection
    m.add_to_collection(batch=True)
  File "/app/plextraktsync/media.py", line 89, in add_to_collection
    self.trakt_api.add_to_collection(self.trakt, self.plex, batch=batch)
  File "/app/plextraktsync/trakt_api.py", line 211, in add_to_collection
    item = dict(**m.ids, **pm.to_json())
  File "/app/plextraktsync/plex_api.py", line 390, in to_json
    "collected_at": timestamp(self.collected_at),
  File "/app/plextraktsync/plex_api.py", line 227, in collected_at
    return self.date_value(self.item.addedAt)
  File "/app/plextraktsync/plex_api.py", line 377, in date_value
    raise ValueError("Value can't be None")
ValueError: Value can't be None
```

### Expected behavior

The scan to complete successfully.

### Steps to reproduce the behavior

docker run --rm -it -v /media/pts/config:/app/config ghcr.io/taxel/plextraktsync

### Install method

docker-compose

### Version

0.20.1

### Python Version

3.10.5

### Operating System and Version

Debian 10

**Comment by twolaw** [comment]
Can you run script with debug mode (edit config.yml) and read logs (or paste it here) to find which plex episode is processed when it crashes.
Then run **inspect** command to this plex episode.

**Comment by rasmuslp** [comment]
Running with debug, the last lines of the log are
```
2022-06-21 17:08:04,730 DEBUG[requests_cache.session]:Skipping cache write for URL: https://server.address.plex.direct:32400/library/metadata/37244?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1
2022-06-21 17:08:04,733 INFO[PlexTraktSync]:Adding to collection: <tmdb:65848:Episode:37244:Veronica-Mars-s01e12>
```
However, I'm not sure how to actually use the `inspect` command. I assume I pass `37244`, like shown below, but no matter what I put after `inspect`, it fails with the same error:
```
$ docker run --rm -it -v /media/pts/config:/app/config ghcr.io/taxel/plextraktsync inspect 37244
PlexTraktSync [0.20.1]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/app/plextraktsync/__main__.py", line 18, in <module>
    cli()
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/app/plextraktsync/cli.py", line 28, in wrap
    cmd(*args, **kwargs)
  File "/app/plextraktsync/commands/inspect.py", line 99, in inspect
    from plextraktsync.util.expand_id import expand_id
ModuleNotFoundError: No module named 'plextraktsync.util'
```

**Comment by twolaw** [comment]
An issue in another issue 😢 

As an alterlative, you can [check the XML](https://support.plex.tv/articles/201998867-investigate-media-information-and-formats/) of this _Veronica Mars_ episode in Plex. The idea is to see if **addedAt** is populated. Compare it with other episodes to see if there is a problem in Plex.
As an example here is one of mine :
`duration="3471040" originallyAvailableAt="2020-03-27" addedAt="1653572877" updatedAt="1654240511"`

**Comment by glensc** [comment]
util module packaging for docker should be fixed in 0.20.2:
- https://github.com/Taxel/PlexTraktSync/pull/950

**Comment by rasmuslp** [comment]
> util module packaging for docker should be fixed in 0.20.2:
> 
> * [Fix util packaging for docker #950](https://github.com/Taxel/PlexTraktSync/pull/950)

Confirmed as working, thank you.

**Comment by rasmuslp** [comment]
I wanted to see if I could make it complete the sync, which unfortunately means that now I can no longer reproduce the issue :/

I tried different things like upgrading to the latest Plex version (was a few releases behind) and refreshing metadata. But that didn't change anything.
In the end, I had to remove the series completely, and empty the Plex trash for the sync to complete.
However, after readding the series, now version `0.20.1` no longer crashes.

I'm not sure what the difference in data could be, but I guess things like 'plays' are only cleared when completely removing an item.
